### PR TITLE
#273 set timer as slider

### LINKED ISSUE #273: Edit task - Set the timer - as slider
This is MVP version of #86 but instead of using wheel input, we can use slider from 1 to 60 minutes

AC: 
- Set slider from 1 to 60 minutes in task create/edit view
- Save this info on task save

### PR DESCRIPTION
**Pull request by Cervera96**
Resolves #273

### CODE REVIEW

File: src/screens/planItemActivity/TimeSlider.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import { palette } from 'styles';
+
+interface Props {
+  min: number;
+  max: number;
+  closeModal?: () => void;
+  onConfirm?: (time: number) => void;
+}
+
+export const TimeSlider: SFC<Props> = ({ min, max, closeModal, onConfirm }) => {
```
Comment (bklukaczewski) [inline src/screens/planItemActivity/TimeSlider.tsx]: isn't `SFC` deprecated in favor of `FC`? With react hooks it's no longer "stateless" ;)

**Comment by bklukaczewski** [comment]
@Cervera96 take a look how you can automatically close an issue by merging the PR
https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

I've added 
> Resolves #273

to the description of the PR. Now when we merge this PR, the source issue will be automatically closed

File: src/components/Slider.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React, { SFC } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { default as SliderRN } from '@react-native-community/slider'; // tslint:disable-line
```
Comment (bklukaczewski) [inline src/components/Slider.tsx]: I think we can remove this rule from TSlint completely

Comment (Cervera96) [reply src/components/Slider.tsx]: I've removed it in this PR https://github.com/autyzm-pg/friendly-plans-react/pull/278

**Pull request merged**